src/guides/v2.3/graphql/tutorials/checkout/checkout-place-order.md

```diff
@@ -18,12 +18,11 @@ The `placeOrder` mutation places an order.
 
 `{ CART_ID }` is the unique shopping cart ID from [Step 2. Create empty cart]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-add-product-to-cart.html).
 
-**Request:**
+Send the customer's authorization token in the `Authorization` parameter of the header. See [Authorization tokens]({{page.baseurl}}/graphql/authorization-tokens.html) for more information.
 
-{:.bs-callout .bs-callout-info}
-For logged-in customers, send the customer's authorization token in the `Authorization` parameter of the header. See [Authorization tokens]({{page.baseurl}}/graphql/authorization-tokens.html) for more information.
+**Request:**
 
-```text
+```graphql
```

Comment:
In the last line of the topic, the right parenthesis is missing.

Reply:
Done.